### PR TITLE
Update GP-Latent.ipynb to v4

### PR DESCRIPTION
**Pull request by bwengals**
# Update GP-Latent.ipynb to V4

This notebook was previously merged (see discussion in #371) but was overwritten.  This PR re-adds it.

### CODE REVIEW

File: myst_nbs/gaussian_processes/GP-Latent.myst.md [hunk 1]
```diff
@@ -6,50 +6,41 @@ jupytext:
...
 ---
 
-# Latent Variable Implementation
+(gp_latent)=
+# Gaussian Processes: Latent Variable Implementation
 
-The `gp.Latent` class is a direct implementation of a GP.  It is called "Latent" because the underlying function values are treated as latent variables.  It has a `prior` method, and a `conditional` method.  Given a mean and covariance function, the function $f(x)$ is modeled as,
+:::{post} Jun 4, 2022
+:tags: gaussian processes, time series
+:category: reference, intermediate
```
Comment (OriolAbril) [inline myst_nbs/gaussian_processes/GP-Latent.myst.md:19]: Also, this is perfect, thanks

File: myst_nbs/gaussian_processes/GP-Latent.myst.md [hunk 2]
```diff
@@ -6,50 +6,41 @@ jupytext:
...
+:author: Bill Engels
+:::
+
++++
+
+The `gp.Latent` class is a direct implementation of a Gaussian process without approximation.  Given a mean and covariance function, we can place a prior on the function $f(x)$,
```
[the remark this answers is not in the document]
Comment (bwengals) [reply myst_nbs/gaussian_processes/GP-Latent.myst.md:25]: That is very cool.  Is this the page to read about how this works? https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-ref-role

**Comment by bwengals** [comment]
<p>Ah yup, added</p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/413/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by bwengals** [comment]
<p>Yup it does.  <code>plot_gp_dist</code> really needs to be updated to work with <code>InferenceData</code> objects.    Really like <code>az.extract</code>, though I have to brag that I'm pretty fast now at typing idata.posterior.thing.stack(sample=['chain', 'draw'])  </p>

---

 <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/413/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: myst_nbs/gaussian_processes/GP-Latent.myst.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (OriolAbril) [reply myst_nbs/gaussian_processes/GP-Latent.myst.md]: yes, this is a good reference

File: myst_nbs/gaussian_processes/GP-Latent.myst.md [hunk 3]
```diff
@@ -6,50 +6,41 @@ jupytext:
...
-The `gp.Latent` class is a direct implementation of a GP.  It is called "Latent" because the underlying function values are treated as latent variables.  It has a `prior` method, and a `conditional` method.  Given a mean and covariance function, the function $f(x)$ is modeled as,
+:::{post} Sept 28, 2022
+:tags: gaussian processes, time series
+:category: reference, intermediate
+:author: Bill Engels
+:::
+
++++
+
+The {class}`gp.Latent` class is a direct implementation of a Gaussian process without approximation.  Given a mean and covariance function, we can place a prior on the function $f(x)$,
```
Comment (OriolAbril) [inline myst_nbs/gaussian_processes/GP-Latent.myst.md:25]: The target should be `pymc.gp.Latent`. If you want the text to be `gp.Latent` you need to do `` {class}`gp.Latent <pymc.gp.Latent>` ``

Comment (bwengals) [reply myst_nbs/gaussian_processes/GP-Latent.myst.md]: whoops, I missed that.  Thanks for the tip

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**